### ISSUE TITLE
Feature: Support for AsyncAPI version 3.0.0

### ISSUE THREAD

**Issue by KevinD-87** [neuroglia-io/asyncapi#31]
## Description
I noticed the YAML output indicates the AsyncAPI version used is version 2.6.0. Are there any plans for supporting version 3.0.0?

## Alternatives
N/A

## Additional info
N/A

**Comment by cdavernas** [comment]
Yes, I do intend on implementing 3.0.0, but my hands are kinduv full for the time being. Will keep you posted whenever I'll have time to start working on it!

**Comment by cdavernas** [comment]
Cfr. https://github.com/neuroglia-io/asyncapi/discussions/28

**Comment by KevinD-87** [comment]
Awesome, thanks! Sorry, I didn't know about the discussion section where this was already asked.
